pkg/kubelet/prober/prober.go

```diff
@@ -80,6 +80,20 @@ func newProber(
 	}
 }
 
+// recordContainerEvent should be used by the prober for all container related events.
+func (pb *prober) recordContainerEvent(pod *v1.Pod, container *v1.Container, containerID kubecontainer.ContainerID, eventType, reason, message string, args ...interface{}) {
+	var err error
+	ref, hasRef := pb.refManager.GetRef(containerID)
+	if !hasRef {
+		ref, err = kubecontainer.GenerateContainerRef(pod, container)
+	}
+	if err != nil {
```

Comment:
nit: move this into the `!hasRef` block above

Reply:
Indeed, good point! Done. Thanks for the review.